src/Microsoft.OData.Core/Value/ODataBinaryStreamValue.cs

```diff
@@ -20,7 +20,7 @@ public sealed class ODataBinaryStreamValue : ODataValue
         /// </summary>
         /// <param name="stream">Input stream</param>
         public ODataBinaryStreamValue(Stream stream) :
-            this(stream, true) // Leaves the stream open by default, for backward compatibility.
+            this(stream, leaveOpen: false)

```

Comment:
>leaveOpen: false [](http://example.com/codeflow?start=25&length=16)

are there no tests that need to be updated by changing this default? can we add tests to verify this default?